packages/ui-voip/src/components/VoipPopup/views/VoipOngoingView.tsx

```diff
@@ -63,6 +64,8 @@ const VoipOngoingView = ({ session, position }: VoipOngoingViewProps) => {
 			</Footer>
 		</Container>
 	);
-};
+});
+
+VoipOngoingView.displayName = 'VoipOngoingView';
```

Comment:
Using a named function as the render function in `forwardRef()` does the same trick.